dev/bots/custom_rules/analyze.dart

```diff
@@ -0,0 +1,78 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' show Directory;
+
+import 'package:analyzer/dart/analysis/analysis_context.dart';
```

Comment:
Update the pubspec to make analyzer a direct dependency instead of an auto-generated transitive one?